programs-remote-listings/templates/shortcode-programs-single.php

```diff
@@ -4,12 +4,17 @@
 global $shortcode_atts;
 global $rs_the_program;
 $options = get_option('rs_remote_settings');
+
+$date_title = ! empty($rs_the_program->date_title) ? $rs_the_program->date_title : 'Date &amp; Time Details:';
+$location_title = ! empty($rs_the_program->location_title) ? $rs_the_program->location_title : 'Location:';
+$address_title = ! empty($rs_the_program->address_title) ? $rs_the_program->address_title : 'Address:';
+
 ?>
 
 <?php
 if (is_array($shortcode_atts)) extract($shortcode_atts); ?>
 
-<article class="single-program">
+<article class="single-program" id="rs-single-program-id-<?php echo $rs_the_program->ID; ?>">
```

Comment:
You will probably want to do the same variable setting as above for date_title, location_title, and address_title given the potential for this to throw a warning.

Reply:
you mean for the values like $rs_the_program->price_details 
??